backend/src/schema/resolvers/passwordReset.spec.js

```diff
@@ -24,7 +24,8 @@ beforeEach(() => {
   variables = {}
 })
 
-beforeAll(() => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-18T07:12:40Z" title="Thursday, June 18th 2020, 9:12:40 am +02:00">Jun 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,6 +25,7 @@ beforeEach(() => {
 })
 
 beforeAll(() => {
+  cleanDatabase()
```


Nice you added this! 😍

But the `await` is missing:
```js
await cleanDatabase()
```
would be correct.